ci/docker/runtime_functions.sh

```diff
@@ -278,8 +278,6 @@ build_ubuntu_gpu_cmake() {
     cmake \
         -DUSE_CUDA=1               \
         -DUSE_CUDNN=1              \
-        -DUSE_MKLML_MKL=0          \
-        -DUSE_MKLDNN=0             \
```

Comment:
I think this one was to explicitely test without MKLDNN.

Reply:
I see the value in explicitly turning off MKL, but what we want to do here is test the default build.  I think CI should reflect how users are likely to actually use a project, and I don't think they're likely to explicitly turn off all the settings they're not using.  We can certainly do both if you prefer, but in my mind it's more important to ensure basic builds work.

If you feel strongly these should stay in I'll amend the commit, after this patch it should work in either case.

Reply:
This case is handled by build_ubuntu_gpu_cmake_mkldnn. The idea here is to create a variety of environments and different build configurations.

Reply:
build_ubuntu_gpu_cmake_mkldnn explicitly turns _on_ MKL, which is fine because if you're opting into a feature you know what that feature is.  My intent here was to test the default behaviour (what a user is likely to do).  I.e. what happens when  you don't explicitly change any other option other than CUDA.